packages/lab/stories/banner/banner.stories.tsx

```diff
@@ -76,9 +76,12 @@ export const MultipleLines = (props: BannerProps) => {
         <Banner {...props}>
           <BannerContent>
             <div>
-              Our guidance for hyphen and dash usage differs from that of the
-              “AP Stylebook” and is aligned with the “J.P. Morgan Brand
-              Guidelines” (also known as the Masterbrand guide).
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```

Comment:
We should be trying to avoid using lorem ipsum. Is there a better example text we can use here?